pkgs/test/nixpkgs-check-by-name/src/nixpkgs_problem.rs

```diff
@@ -213,6 +227,53 @@ impl fmt::Display for NixpkgsProblem {
                     subpath.display(),
                     text,
                 ),
+            NixpkgsProblem::InternalCallPackageUsed { attr_name } =>
+                write!(
+                    f,
+                    "pkgs.{attr_name}: This attribute is defined using `_internalCallByNamePackageFile`, which is an internal function not intended for manual use.",
+                ),
+            NixpkgsProblem::MovedOutOfByName { package_name, call_package_path, empty_arg } => {
+                let call_package_arg =
+                    if let Some(path) = &call_package_path {
+                        format!("./{}", path.display())
+                    } else {
+                        "...".into()
+                    };
+                if *empty_arg {
+                    write!(
+                        f,
+                        "pkgs.{package_name}: This top-level package was previously defined in {}, but is now manually defined as `callPackage {call_package_arg} {{ }}` (e.g. in `pkgs/top-level/all-packages.nix`). This should not be necessary, please move the package back and remove the manual `callPackage`.",
```

Comment:
Comma splice in the last sentence. Replace comma with semicolon.

> This should not be necessary; please move the package back and remove the manual `callPackage`.

Reply:
Semicolon feels a bit weird, I just removed the "This should not be necessary" part. That part of the sentence itself shouldn't be necessary. If we discover cases where it's still necessary we can reconsider